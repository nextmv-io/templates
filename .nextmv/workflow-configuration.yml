apps:
  - name: python
    type: python
    app_id: python
    description: Use Python to start writing your optimization model.
  - name: java
    type: java
    app_id: java
    description: Use Java to start writing your optimization model.
  - name: go
    type: go
    app_id: go
    description: Use Go to start writing your optimization model.

# Main supported language versions.
language-support:
  go:
    version: 1.22
  python:
    version: 3.11
  java:
    version: 21
    distribution: temurin
