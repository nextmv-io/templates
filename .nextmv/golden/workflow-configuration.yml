apps:
  - name: python
    language: python
    language-version: 3.11
  - name: go
    language: go # we always use our main version
  - name: java
    language: java
    language-version: 21
    distribution: temurin
